README.md

```diff
@@ -10,3 +10,52 @@ A blogging platform for the creative at heart.
 ## Documentation
 
 Comprehensive documentation for the API is hosted [here](https://ah-kg-avengers-backend-staging.herokuapp.com/swagger).
+
+## Features
+
+- Users can create accounts to store their data.
+- Users can create, edit and delete articles.
+- Users can read other authors' articles.
+- Users can follow other authors and share these articles on Social Platforms.
+
+## Tools
+
+Tools used for development are;
+
+- Documentation : [Swagger](https://swagger.io/)
+- Framework: [http://react/](https://reactjs.org/)
+- Code Editor/IDE: [VSCode](https://code.visualstudio.com)
+- Programming language: [JavaScript(ES6)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/)
+- Testing environment: [Jest](https://www.getpostman.com)
+
+## Getting Started
+
+1. After cloning the repository from [here](https://github.com/andela/ah-kgl-avengers-frontend), install requirements by running this command in the root of the repo folder
+
+```sh
+    $ npm install
+```
+
+2. Make a copy of `.env.sample` and rename it to `.env`. Edit the file replacing the respective configuration values.
+
+3. Run the following command to start the API server:
+
+```sh
+    $ npm start
+```
+
+## Running the tests
+
+To be updated
+
+## Contributors
+
+Many thanks to `Ezrqn Kemboi` and `Bridget Mendoza`.
+
+### Key Contributors
+
+- Eric Shema
```

Comment:
It should be better if you put URL references direct to the contributor's git page

Reply:
Adding to what @aaronsekisambu said, can you convert the code to ES6 since you added babel for compiler.

Reply:
@niyobobo @aaronsekisambu  Changing this file to ES6 would result in renaming the file to `webpack.config.babel.js` for babel to know that it has to transpile this file too. I think that the name is not clear and ES5 in this file only can't be bad! What do you think?

Reply:
@avpaul Technically I have no answer to this. Maybe @me-x-mi can help here or @kipropbrian

Reply:
I think for my side we can stay with those codes in ES5 as they are, It is one file and does not affect the codebase.

Reply:
@avpaul tried to check on ```javascript webpack.config.babel.js``` so that I can write Es6 in the file though run into several errors including getting Json and babel version issues as babel registry was required. 
Is there any other person who has done research on the same or has tried to change the file to Es6? 
Please let us know. Cc.. @shemaeric @kipropbrian @avpaul @aaronsekisambu @niyobobo @fridolinho .

Reply:
Now it's fixed 👏 👏  I had to install `@babel/register` and change the config file name to `webpack.config.babel.js` Now our config is in ES6, thanks for requesting the change 👍